sequoia/http.py

```diff
@@ -77,6 +81,10 @@ def __init__(self, auth, session=None, proxies=None, user_agent=None, get_delay=
 
         self.request_timeout = request_timeout or env.DEFAULT_REQUEST_TIMEOUT_SECONDS
 
+    def _remove_none_to_avoid_infinite_retries(self):
+        if 'max_time' in self.backoff_strategy and not self.backoff_strategy['max_time']:
+            del self.backoff_strategy['max_time']
```

Comment:
Shouldn't be better set to default value instead of deleting the `max_time` in case of None, wdyt?